.gitattributes

```diff
@@ -0,0 +1,7 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/composer.lock export-ignore
+/docs export-ignore
```

Comment:
Sure about this one?

Reply:
yes. I am:

> This makes composer package contain only needed files:
> 
> - readme, license
> - php source

you can always install --prefer-source to get all files, or browse online rest of the files. documentation in vendor did is not required for package to be used.